Lib/test/test_inspect.py

```diff
@@ -620,6 +620,13 @@ def __repr__(self):
         with self.assertRaises(TypeError):
             inspect.getfile(er)
 
+    def test_getfile_implicit_namespace_package(self):
+        import inspect_implicit_namespace_pkg
```

Comment:
You could use some of existing namespace packages in test_import or /test_importlib.

And for now tests are failing.